packages/promise/promise.1.1.2/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+
+synopsis: "Native implementation of a JS promise binding"
+
+version: "1.1.2"
+license: "MIT"
+homepage: "https://github.com/aantron/promise"
+doc: "https://github.com/aantron/promise"
+bug-reports: "https://github.com/aantron/promise/issues"
+
+authors: "Anton Bachin <antonbachin@yahoo.com>"
+maintainer: "Anton Bachin <antonbachin@yahoo.com>"
+dev-repo: "git+https://github.com/aantron/promise.git"
+
+depends: [
+  "dune"
+  "ocaml"
+  "reason" {build & >= "3.3.2"}
+  "result"
```

Comment:
I see the dependency here, maybe it needs to be linked explicitly or you need to open the module?